var/spack/repos/builtin/packages/parallel-netcdf/package.py

```diff
@@ -122,34 +125,37 @@ def configure_args(self):
 
         flags = {"CFLAGS": [], "CXXFLAGS": [], "FFLAGS": [], "FCFLAGS": []}
 
-        if "+pic" in self.spec:
+        if self.spec.satisfies("+pic"):
             flags["CFLAGS"].append(self.compiler.cc_pic_flag)
             flags["CXXFLAGS"].append(self.compiler.cxx_pic_flag)
             flags["FFLAGS"].append(self.compiler.f77_pic_flag)
             flags["FCFLAGS"].append(self.compiler.fc_pic_flag)
 
         # https://github.com/Parallel-NetCDF/PnetCDF/issues/61
-        if self.spec.satisfies("%gcc@10:"):
+        if self.spec.satisfies("@:1.12") and self.spec.satisfies("%gcc@10:"):
```

Comment:
It looks like the issue is addressed in version `1.12.2` (see [here](https://github.com/Parallel-NetCDF/PnetCDF/commit/fc7af03a87c796625258ee183f21b8e74974ab58)). I would also combine the specs:
```suggestion
        if self.spec.satisfies("@:1.12.1%gcc@10:"):
```

Reply:
Done! Thanks for catching that.